test/github-issues/10456/issue-10456.ts

```diff
@@ -0,0 +1,71 @@
+import "reflect-metadata"
+import {
+    createTestingConnections,
+    closeTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { DataSource } from "../../../src/data-source/DataSource"
+import { expect } from "chai"
+import { Example } from "./entity/example"
+
+// values from https://github.com/obartunov/sqljsondoc/blob/master/jsonpath.md#jsonpath-in-postgresql
+const validJsonPaths = [
+    "$", // the whole JSON document (context item)
+    "$foo", // variable "foo"
+    '"bar"', // string literal
+    "12.345", // numeric literal
+    "true", // boolean literal
+    "null", // null
+    "$.floor", // field accessor on $
+    "$.floor[*]", // the same, followed by wildcard array accessor
+
+    // complex path with filters and variables
+    "$.floor[*] ? (@.level < $max_level).apt[*] ? (@.area > $min_area).no",
+
+    // arithmetic expressions:
+    "-$.a[*]", // unary
+    "$.a + 3", // binary
+    "2 * $.a - (3 / $.b + $x.y)", // complex expression with variables
+
+    // parenthesized expression used as starting element of a path,
+    // followed by two item methods ".abs()" and ".ceil()"
+    "($ + 1).abs().ceil()",
+
+    "$.floor[*].apt[*].area > 10",
+]
+
+describe("github issues > #10456 adds jsonpath column type", () => {
+    let connections: DataSource[]
+    before(
+        async () =>
+            (connections = await createTestingConnections({
+                entities: [Example],
+                enabledDrivers: ["postgres"],
+                schemaCreate: true,
+                dropSchema: true,
+            })),
+    )
+
+    beforeEach(() => reloadTestingDatabases(connections))
+    after(() => closeTestingConnections(connections))
+
+    it("should insert and retrieve jsonpath values as strings", () =>
```

Comment:
I think all those tests should be implemented in `test/functional/database-schema/column-types/postgres/column-types-postgres.ts`. This is not a bug we are dealing with here, rather new feature. It would be super hard to find those tests later when the burried in `issue-10456.ts`